tests/test_fixtures/rule_based_profiler/example_notebooks/BasicExample_RBP_Instantiation_and_running.ipynb

```diff
@@ -0,0 +1,1778 @@
+   "source": [
+    "#### `MetricMultiBatchParameterBuilder`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `MetricMultiBatchParameterBuilder` contains a single or multi-batch implementation for obtaining a resolved (evaluated) metric, using `domain_kwargs`, `value_kwargs`, and `metric_name` as arguments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "numeric_range_parameter_builder: MetricMultiBatchParameterBuilder = (\n",
+    "    MetricMultiBatchParameterBuilder(\n",
+    "        data_context=data_context,\n",
+    "        batch_request=multi_batch_batch_request, # we are passing in our multi_batch_batchr_request here\n",
+    "        metric_name=\"column.min\",\n",
+    "        metric_domain_kwargs=domain.domain_kwargs,\n",
+    "        name=\"my_column_min\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain: Domain = Domain(domain_type=MetricDomainTypes.COLUMN, domain_kwargs = {'column': 'total_amount'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "69c8f68ef91547f2a0d813b5804e10e2",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Calculating Metrics:   0%|          | 0/48 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'parameter': {'parameter': {'my_column_min': {'value': [-19.8, -57.3, -6.8, -63.06, -11.8, -6.8, -30.6, -16.8, -4.3, -100.8, -12.8, -80.3], 'details': {'metric_configuration': {'metric_name': 'column.min', 'domain_kwargs': {'column': 'total_amount'}, 'metric_value_kwargs': None, 'metric_dependencies': None}, 'num_batches': 12}}}}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "numeric_range_parameter_builder.build_parameters(domain=domain, parameter_container=parameter_container)\n",
+    "# we check the parameter container\n",
+    "print(parameter_container.parameter_nodes)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`my_column_min[value]` now contains a list of 12 values, which are the minimum values the `total_amount` column for each of the 12 batches associated with 2018 `taxi_data` data. If we were to use the latest (December) value in a `ExpectationConfigurationBuilder`, it would be accessible through the fully-qualified parameter: `$parameter.my_column_min.value[-1]`"
```

Comment:
How would you specify using the minimum value from the 12 values?

Reply:
This statement was a bit-misleading. Instead of the `ExpectationConfigurationBuilder` using the most recent value only, we would pass along the entire parameter : `$parameter.my_column_min.value`.  

To answer your question, I think for other ParameterBuilders like `NumericMetricRangeMultiBatchParameterBuilder`, we get a `min_value`, `max_value` that would be accessible as `$parameter.my_column_min.value.min_value`